tests/src/test/java/alluxio/client/cli/fs/command/PersistCommandTest.java

```diff
@@ -195,4 +192,24 @@ public void persistWithWildCard() throws Exception {
     checkFilePersisted(new AlluxioURI(filePath2), 20);
     checkFilePersisted(new AlluxioURI(filePath3), 30);
   }
+
+  @Test
+  public void persistParallel() throws Exception {
```

Comment:
would be nice to generalize this to persist N files with M threads, then try a few different variations of N and M, e.g. N < M, N == M, N > M, N % M != 0